src/main/java/ca/team2706/frc/robot/config/FluidConstant.java

```diff
@@ -37,19 +36,18 @@
         this.value = initialValue;
         this.deployedValue = initialValue;
 
-        Robot.setOnStateChange(this::addNTEntry);
+        Robot.setOnStateChange(robotState -> addNTEntry());

```

Comment:
Why not check if the robot is initialized here, and only call addNTEntry once?

Reply:
Good suggestion.